### PR TITLE
Add quotes to the beginnings of chapters

### PR DESCRIPTION
**Pull request by hdashnow**
Addresses #243, except ch5
(also removed some random spaces)

### CODE REVIEW

**Comment by hdashnow** [comment]
I'm not sure how I managed to introduce that wercker build error. Any ideas?

File: markdown/ch1.markdown [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Elegant NumPy: The Foundation of Scientific Python
 
+> "[NumPy] is everywhere. It is all around us. Even now, in this very room.
+> You can see it when you look out your window or when you turn on your
+> television. You can feel it when you go to work... when you go to church...
+> when you pay your taxes."
```
Comment (jni) [inline markdown/ch1.markdown]: We don't use quotes when already in quote mode with `>`. See ch8 for a template!

Comment (jni) [inline markdown/ch1.markdown]: I think leave out the [] around NumPy. That makes it even better. (Deadpan. =D)

File: markdown/ch1.markdown [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+>
+> -- Morpheus, *The Matrix*
```
Comment (jni) [inline markdown/ch1.markdown]: — (alt-shift-'-' on Mac)

File: markdown/ch3.markdown [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Networks of Image Regions with ndimage
 
+> "Tyger Tyger, burning bright,
```
Comment (jni) [inline markdown/ch3.markdown]: You can do newlines in Markdown without starting a new paragraph: add two spaces at the end. Though there might have been an issue with that in the HTML conversion — @stefanv?

File: markdown/ch3.markdown [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+>
+> In the forests of the night;
+>
+> What immortal hand or eye,
+>
+> Could frame thy fearful symmetry?"
+>
+> -- William Blake, *The Tyger,*
```
Comment (jni) [inline markdown/ch3.markdown]: Is the second comma necessary?

**Comment by jni** [comment]
@hdashnow not sure about Wercker! I tried to reproduce locally but getting a bs4 lxml error even though I have lxml installed. @stefanv?

@hdashnow in the meantime, I had a few suggestions.

File: markdown/ch1.markdown (hunk 1 above)
Comment (hdashnow) [inline markdown/ch1.markdown]: `[]` are staying in 😛

**Comment by hdashnow** [comment]
@jni better?

**Comment by jni** [comment]
Yep! Thanks for fixing mine! ;) Now we just need @stefanv to comment on the build. =\

**Pull request merged**